rules/no-hello/rule.md

```diff
@@ -63,82 +65,18 @@ Bad example - They're left waiting for you to ask a question
 :::
 
 Asking your question before getting any reply allows **asynchronous communication**. If the other party is away, and you leave before they come back, they can still answer your question, instead of just staring at a "Hello" and wondering what they missed.
 
-There are 2 situations where you want to message someone: 
-
-* Quick simple questions 
-* More complex ones that need a discussion
-
-## Simple - Make your questions easy to respond to
+## Tip - Make your questions easy to respond to
 
-For simple questions, try to phrase it so they have easy options, then you can get away with asking these over text chat. 
-
-Try to give 2 options and ask them to thumbs up 👍 their preferred option. That makes it as simple as possible to respond so you can even catch people who are extremely busy!
+For simple questions, try to give 2 or more options and ask them to thumbs up 👍 their preferred option. That makes it as simple as possible to respond so you can even catch people who are extremely busy!
 
 ![Figure: ✅ Good example - The question and options are easy for a busy person to action ](no-hello.jpg)
 
-## Complex - start a call warm, not cold
-
-If your conversation requires a discussion, then it's time to initiate a call. If the conversation has history, like a PBI or email, then it's important to include that context. That way the other person can prioritise your call. 
-
-For example, if you mention that you want to talk about an email from several years ago they might think it isn't that important.
-
-Conversely, if you mention the production server is down, they will almost definitely answer you!
-
-Other reasons for this:
-
-* It is important to go the extra mile when you are asking someone else for something… so be as considerate as possible
-* It puts you into their chat history, so even if they decline the call, they can easily write back to you to explain why
-* Giving someone even a 1 minute warning is a good idea even if you see their Teams Status is on green ‘available’. Remember it is not easy to always change their status to ‘busy’ - they could be on a mobile phone, on a WhatsApp call, in a physical meeting, etc.
-
-Also, everyone likes breathing space, so try to give about a minute's warning for any call.
-
-:::greybox
-Please see email subject: **{{Email subject}}**  
-
-I'll call in 1 minute.
-:::
-:::good
-Good example - Includes a task, and gives context
-:::
-
-:::greybox
-Calling in 1 minute. See the PBI:
-
-**{{URL}}**
-:::
-:::good
-Good example - Includes context and doesn't leave anyone waiting
-:::
-
-**Tip:** For the example above, PBIs often have long URLs. Break a line for better readability.
-
-### Using the Share to Teams option via Outlook
-
-![Figure: Share to Teams button](share-to-teams-outlook-button.png)
-
-:::good
-![Figure: Good example - Includes a task, and gives context by sharing the email to a teams chat using the Share to Teams option via Outlook](share-to-teams.png)
-:::
```

Comment:
Sam - Leave this point as it gives context and should be on the No Hello rule
-a

Reply:
Hey Adam,

I've added it back. :)